### DISCUSSION TITLE
How to use relayer network outside of the US?

### DISCUSSION THREAD

**Discussion by conorpp** [WalletConnect/walletconnect-monorepo#2063] [Developer Support] [answered]
A number of Dapps restrict access from the US.  The default wallet connect relayer network appears to be in the US which causes some issues with restrictive Dapps.  

This wallet connect example used to support different relayer regions: https://github.com/WalletConnect/web-examples/tree/main/wallets/react-wallet-auth

```
  {
    value: 'wss://us-east-1.relay.walletconnect.com/',
    label: 'US'
  },
  {
    value: 'wss://eu-central-1.relay.walletconnect.com/',
    label: 'EU'
  },
  {
    value: 'wss://ap-southeast-1.relay.walletconnect.com/',
    label: 'Asia Pacific'
  }
```

Are these still working?  I'm getting websocket connection issues when trying to create a `SignClient` using any of these endpoints.  Is there another way to use a regional relayer network?

**Comment by finessevanes** [comment, marked as answer]
Try using `wss://relay.walletconnect.org`
Docs: https://docs.walletconnect.com/2.0/advanced/faq
